Process.h

```diff
@@ -324,6 +327,10 @@ static inline bool Process_isThread(const Process* this) {
    return Process_isUserlandThread(this) || Process_isKernelThread(this);
 }
 
+static inline bool Process_isRunningInContainer(const Process* this) {
+   return this->isRunningInContainer;
+}
+
```

Comment:
Why have this extra function/indirection when the only caller could just use the flag directly?

Reply:
> Feel free to clean up the history on your PR as much as you like

Cool. I'll clean up the commit history and push it again.

Reply:
`Process_isKernelThread` and `Process_isUserlandThread` are there too. Since, those two features also have this option of hiding/displaying the option, I thought I would keep `Process_isRunningInContainer` there as well so that all three remain together. Shall I remove `Process_isRunningInContainer`?

Reply:
Although, now that I think about it, it might not be necessary since there is only one caller.